src/test/resources/unit/cyclonedx/valid-bom-1.6.json

```diff
@@ -32,7 +32,11 @@
     ],
     "component": {
       "type": "application",
-      "author": "Acme Super Heros",
```

Comment:
Consider creating a separate BOM file instead, and using that in a new test in `BomUploadProcessingTaskTest`. Ideally we would want to verify that `Component#authors` is populated correctly when either `authors` or `author` is provided in the BOM. Even better, we should have a test case that demonstrates what happens when both `author` *and* `authors` are set in the BOM.